frontend/src/App.js

```diff
@@ -62,9 +56,17 @@ class App extends Component {
                             exact
                             path="/admin/events"
                             render={() => (
-                                <AdminEvents 
-                                events={this.state.events} 
-                                deleteEvent={this.toDelete}/>
+                                <AdminEvents
+                                    events={this.state.events}
+                                    deleteEvent={this.toDelete}
+                                />
+                            )}
+                        />
+                        <Route
+                            exact
+                            path="/admin/floaters"
+                            render={() => (
+                                <Floaters mentors={this.state.mentors} />
```

Comment:
There is no `mentors` entry in `state`, and there is no need in passing mentors to the component, as it fetches mentors inside. You need or fetch here, and don't fetch inside Floaters, or remove this property.

Reply:
I'd recommend to be consistent with events, but it will require refactoring.